nio/api.py

```diff
@@ -232,6 +232,26 @@ def login(
 
         return "POST", path, Api.to_json(content_dict)
 
+    @staticmethod
+    def logout(
+        access_token,     # type: str
+    ):
+        """Logout the session.
+
+        Returns nothing.
+
+        Args:
+            access_token (str): the access token to be used with the request.
+        """
+        query_parameters = {"access_token": access_token}
+
+        content_dict = {}
```

Comment:
Mypy is annoyingly complaining about this. Adding a `#  type: Dict` silences it.